index.html

```diff
@@ -1 +1 @@
-Hi, my name is not Jon.
+Hi, my name is Jon.
```

Comment:
You should spell your name, John, like normal people.

Reply:
Define normal